README.md

```diff
@@ -98,6 +98,28 @@ It's pretty strait forward.
 
 If you want, you can add a pre-build script to your project to get the manifest's "android:versionName" and set it right into the String.xml file, but that's up to you.
 
+### Use with multi environment setup 
+
+To use with multiple envrionment you can use this setup
+
+First of a Config folder at the root of the solution, containing a Dev and Production folder.
```

Comment:
this is based on our devops setup. On public projects there are no guarantees that $(AppEnvironment) exists nor that it would be defined to Production or Staging.